core/server/models/post.js

```diff
@@ -129,12 +186,12 @@ Post = ghostBookshelf.Model.extend({
 
         // ### Business logic for published_at and published_by
         // If the current status is 'published' and published_at is not set, set it to now
-        if (this.get('status') === 'published' && !this.get('published_at')) {
-            this.set('published_at', new Date());
+        if (newStatus === 'published' && !publishedAt) {
+            this.set('published_at', moment.utc().toDate());
```

Comment:
> so because our server is configured to UTC time, all your dates should be already in UTC in the database (that's what i expect) - tell me if i am wrong.

That's true for most installs, inc all of the ones on Ghost(Pro) & DigitalOcean, but there could well be people who have their servers configured differently out there in the world somewhere.

It's probably best to revert this for now, but look at it as part of adding proper timezone handling - even if it's just to update the docs to say Ghost expects the server time to be UTC :)

> may the next universe be without dates!
Yes please 🍪  !